gulpfile.js

```diff
@@ -109,7 +110,7 @@ gulp.task('build:testjs', function () {
   gulp.src('src/*.js')
     .pipe(replace('{{apiurl}}', TESTAPIURL))
     .pipe(replace('{{cssurl}}', TESTCSSURL))
-    .pipe(replace('{{templates}}', prepareTemplates()))
+    .pipe(replace('\'{{templates}}\'', prepareTemplates()))
```

Comment:
Looks like there is a way to disable rules for a single line http://eslint.org/docs/user-guide/configuring.html#disabling-rules-with-inline-comments. Since this templates line is the only one causing problems in `src` we could disable the rule for this line, and still lint the src code? Thoughts?